lib/rucio/db/sqla/models.py

```diff
@@ -1173,9 +1174,16 @@ class Subscription(BASE, ModelBase, Versioned):
 class Token(BASE, ModelBase):
     """Represents the authentication tokens and their lifetime"""
     __tablename__ = 'tokens'
-    token = Column(String(352))  # account-identity-appid-uuid -> max length: (+ 30 1 255 1 32 1 32)
+    token = Column(String(3072))  # account-identity-appid-uuid -> max length: (+ 30 1 255 1 32 1 32)
     account = Column(InternalAccountString(25))
+    refresh_token = Column(String(315), default=None)
```

Comment:
`oidc_` prefix?

Reply:
see my previous response to the alembic script comment ...